.gitignore

```diff
@@ -56,7 +56,7 @@ CATKIN_IGNORE
 *.pch
 
 # Compiled Dynamic libraries
-*.so
+# *.so
```

Comment:
Note : The Kinova API requires the use of `.so` to load the different functions required to interact with the arm.